cmd/juju/action/listoperations.go

```diff
@@ -177,16 +177,19 @@ func (c *listOperationsCommand) formatTabular(writer io.Writer, value interface{
 
 	printOperations := func(operations []operationLine, utc bool) {
 		for _, line := range operations {
-			w.Print(line.id, line.operation, line.status, line.unit)
-			w.Println(formatTimestamp(line.timestamp, false, c.utc, true))
+			w.Print(line.id, line.status)
+			w.Print(formatTimestamp(line.started, false, c.utc, true))
+			w.Print(formatTimestamp(line.finished, false, c.utc, true))
+			w.Print(strings.Join(line.tasks, ","))
+			w.Println(line.operation)
 		}
 	}
-	w.Println("Id", "Operation", "Status", "Unit", "Time")
+	w.Println("Id", "Status", "Started", "Finished", "Tasks", "Summary")
```

Comment:
A better name for Tasks would be good. Task IDs perhaps is a better label. When I initially ran a single task, I got the number 3 which I thought was the number of tasks :thinking: